rust/src/detect/mod.rs

```diff
@@ -43,6 +44,47 @@ pub trait EnumString<T> {
     fn from_str(s: &str) -> Option<Self> where Self: Sized;
 }
 
+#[repr(u8)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+// endian <big|little|dce>
+pub enum ByteEndian {
+    _EndianNone = 0,
```

Comment:
Could you add a comment why we need this unused value ?

Reply:
It's not needed; will remove.